public/scripts/votingController.js

```diff
@@ -0,0 +1,101 @@
+(function (module) {
+  var voteRepos = {};
+  var member;
+  var voteHistory;
+  var votesArray = ['S J RES 23', 'H R 1927', 'H R 2130', 'H R 1644']; // Specific bills we want to store
+
+  voteRepos.all = [];
+
+  var findBill = function (targetArray, targetBill) {
+    return targetBill.question === 'On Passage' && // 'On Passage' means they're voting to pass the bill.
+           targetArray.includes(targetBill.bill.number);
+  };
+
+  var buildBill = function (proto) { // Takes a bill object and creates a new bill object with specific properties.
+    var voteInfo = {};
```

Comment:
You're doing a lot of heavy lifting here, and could have just used an object constructor.